src/UI/Tooltip.js

```diff
@@ -100,10 +34,69 @@ class Tooltip {
         this.top = null
         document.body.removeChild(this.host)
     }
-    checkPosition = checkPosition.bind(this)
-    updatePosition = updatePosition.bind(this)
-    ensureIsLastInDOM = ensureIsLastInDOM.bind(this)
-    resObs = new ResizeObserver(entries => entries.forEach(this.checkPosition))
+    handleEvent (event) {
```

Comment:
I moved the event handler to _handleEvent_ so that it can be cleanly removed whilst not requiring bind, and storing in a variable. (closures won't ever match)

Also I think it's cleaner when you get much more of them.